kolibri/plugins/coach/assets/src/app.js

```diff
@@ -196,10 +196,10 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXERCISE_RENDER,
-          path: '/:userId/:contentId/exercise-render',
+          path: '/:classId/exercise-render/:userId/:channelId/item/:contentId',
```

Comment:
Discussed this with @DXCanas a moment ago, but we should delete this route and instead use the following three URLs:

* `RECENT_LEARNER_ITEM_DETAILS`
* `TOPIC_LEARNER_ITEM_DETAILS`
* `LEARNER_ITEM_DETAILS`

The URL and "back" link in the top-left corner will be distinct for each of these URLs, but the underlying action logic and views should have shared code.